tikv/mvcc.go

```diff
@@ -265,9 +265,12 @@ func (store *MVCCStore) Commit(req *requestCtx, keys [][]byte, startTS, commitTS
 func (store *MVCCStore) handleLockNotFound(reqCtx *requestCtx, key []byte, startTS, commitTS uint64) error {
 	txn := reqCtx.getDBReader().txn
 	item, err := txn.Get(key)
-	if err != nil {
+	if err != nil && err != badger.ErrKeyNotFound {
 		return errors.Trace(err)
 	}
+	if item == nil {
```

Comment:
Check if err if badger.ErrKeyNotFound  should be more clear.

Reply:
I think `badger.ErrKeyNotFound` is redundant, check if the returned object is nil is a more commonly used pattern.